import_extractor.py

```diff
@@ -0,0 +1,63 @@
+import ast
+import distutils.sysconfig as sysconfig
+import imp
+import pathlib
+import pkgutil
+import os
+import sys
+
+built_in_modules = set(sys.builtin_module_names)
+built_in_modules.update({
+    module
+    for _, module, package in pkgutil.iter_modules()
+    if package is False
+})
+std_lib = sysconfig.get_python_lib(standard_lib=True)
+for top in pathlib.Path(std_lib).iterdir():
+    if top.startswith(('dist-packages', 'site-packages')):
+        continue
+    built_in_modules.add(top.split('.')[0])
+
+# We take the top-folder as input
+location = os.path.abspath(sys.argv[1])
+hint_locations = [pathlib.Path(location, 'Tribler', 'dispersy'),
```

Comment:
I think that all these `hint_locations` can be removed since we are not using any of them anymore 👍

Reply:
Done.